src/argilla/client/api.py

```diff
@@ -206,6 +206,7 @@ def load(
     ids: Optional[List[Union[str, int]]] = None,
     limit: Optional[int] = None,
     id_from: Optional[str] = None,
+    batch_size: Optional[int] = None,
```

Comment:
Maybe it makes sense to have an explicit default value here. 
`batch_size: int = 250`

Reply:
Will do, I'll use 250 throughout 👍

Reply:
Done